.github/workflows/admin.yml

```diff
@@ -42,6 +40,7 @@ jobs:
       - name: Deploy
         env:
           SSH_PRIVATE_KEY: ${{ secrets.SSH_PRIVATE_KEY }}
+          SENTRY_DSN: ${{secrets.SENTRY_DSN}}
```

Comment:
For application env vars, It should be added in docker-compose file and last line of this file.
Review whole command, you will get idea